java-persistence/src/main/java/com/whitecloak/training/inventory/controller/response/CategoryResource.java

```diff
@@ -0,0 +1,36 @@
+package com.whitecloak.training.inventory.controller.response;
+
+import com.whitecloak.training.inventory.persistence.entity.ItemEntity;
+
+import java.util.Set;
+
+public class CategoryResource {
+
+    private long id;
+    private String name;
+    private Set<ItemEntity> items;
```

Comment:
Entity has leaked to the web layer. Remember to contain database-related stuffs to the gateway layer.